src/types/contract_source_metadata/mod.rs

```diff
@@ -72,6 +72,7 @@ pub struct ContractSourceMetadata {
     /// # ;
     /// ```
     // it's a guess it was added as 1.1.0 of nep330, [nep330 1.1.0 standard recording](https://www.youtube.com/watch?v=pBLN9UyE6AA) actually discusses nep351
+    #[serde(default)]
```

Comment:
cross-repo TODO: propagate this change to `near-contract-standards` and `near-api`